crates/oxc_linter/src/lib.rs

```diff
@@ -466,8 +466,11 @@ impl Linter {
                     ));
                 }
             }
-            Err(_err) => {
-                // TODO: report diagnostic
+            Err(err) => {
+                ctx_host.push_diagnostic(Message::new(
+                    OxcDiagnostic::error(format!("Error running JS plugin:\n{err}")),
+                    PossibleFixes::None,
+                ));
```

Comment:
@camc314 Can I just check this is correct? Or can an `OxcDiagnostic::error` get filtered out in some cases?